app/knowledge_panels.py

```diff
@@ -39,3 +41,59 @@ def hunger_game_kp(
             ],
         },
     }
+
+
+def last_edits_kp(
+    facet: str,
+    value: Union[str, None] = None,
+    country: Union[str, None] = None,
+):
+    if facet == "packaging":
+        plural = facet
+    else:
+        plural = facet_plural(facet=facet)
+    query = {
+        "fields": "code,last_editor,last_modified_t",
+        "sort_by": "last_modified_t",
+    }
+    if facet == "country":
+        country_code = country_to_ISO_code(value=value)
+        country = value
+        url = f"https://{country_code}-en.openfoodfacts.org"
+        description = f"{value}"
+    else:
+        if value is not None:
+            url = "https://world.openfoodfacts.org"
+            query[f"{plural}_tags_en"] = value
+            description = f"{value} based for {facet}"
+        if country is not None:
+            country_code = country_to_ISO_code(value=country)
+            url = f"https://{country_code}-en.openfoodfacts.org"
+            description = f"{facet} based for {country}"
+
+    search_url = f"{url}/api/v2/search"
+    description = f"last-edits issues related to {description}"
+    response_API = requests.get(search_url, params=query)
+    data = response_API.json()
+    counts = data["count"]
+    tags = data["products"]
+    html = "\n".join(
+        f'<li>Product with this code {tag["code"]} was last edited by {tag["last_editor"]} has last_modified_tag {tag["last_modified_t"]}</li>'
```

Comment:
In previous request we should also get "product_name" as it gives more information.

Also last_modified_t is a bit difficult to read ! (it's a timestamp), but you can write a function to transform it to a more readable date.

Also try to keep the sentence short. I propose:
`"fior di fragola" (8003100814036) edited by creme on 2022-07-26 10:11`

See [datetime.fromtimestamp](https://docs.python.org/3.9/library/datetime.html#datetime.datetime.fromtimestamp) and [strftime](https://docs.python.org/3.9/library/datetime.html#datetime.datetime.strftime)

There are also nice libraries in python to display dates in a human manner (last hour, about 2 hours ago, last month, etc.)